### DISCUSSION TITLE
Can cards be used like the documentation site?

### DISCUSSION THREAD

**Discussion by lordnynex** [cloudscape-design/components#108] [Q&A] [answered]
I'm attempting to duplicate the documentation sites 'navigational cards'. 

<img width="1155" alt="image" src="https://user-images.githubusercontent.com/1108594/182460640-3be2ae5c-5e97-45b4-9577-26b65a26e968.png">

I'm thinking that thee cards were not created with the `<Cards>` component. Is there any plan to split out the notion of a `<Card>` separately from a `<Cards>` collection? I'm attempting to duplicate it using the collection but I'm noticing what I think are a few issues holding me back. 

* When `CardsProps.CardDefinition.header` is undefined or returns null, there is still an empty card header div/span rendered. Would it make sense to render nothing in this case? 
* It doesn't seem like there a way to disable the padding on the cards themselves.

I'm pretty new to frontend development, so I apologize if my questions are non-sense or miss the mark.

**Comment by YueyingLu** [comment, marked as answer]
Hey @lordnynex ,
thank you for reaching out. 

The component in the screenshot is a custom component built for this use case, not a `<Card />`  component. Let me share [this example implementation](https://codesandbox.io/s/aged-lake-oynldp?file=/src/App.js) with you. We also noticed overflowing corners on the `<Container />` in this use case, which we are currently looking into as part of this [PR](https://github.com/cloudscape-design/components/pull/115).

Let me know if you have any other questions.

Thank you!
Yueying
